src/main/java/io/cdap/plugin/format/DBTableRecordReader.java

```diff
@@ -111,7 +111,7 @@ public StructuredRecord getCurrentValue() throws IOException {
   }
 
   @Override
-  public float getProgress() {
+  public float getProgress() throws IOException {
```

Comment:
why was this added?

Reply:
getLength() method which uses in this method can throw IOException,